tests/modules/align_timestamps/test_align_timestamps_module.py

```diff
@@ -0,0 +1,186 @@
+import json
+import os
+import subprocess as sp
+
+import pytest
+import numpy as np
+
+
+DATA_DIR = os.environ.get(
+    "ECEPHYS_PIPELINE_DATA",
+    os.path.join("/", "allen", "aibs", "informatics", "module_test_data", "ecephys"),
```

Comment:
I assume we are leaving the question of "how do we make this test not depend on Isilon" for a future ticket?